### DISCUSSION TITLE
Any application note to use this framework in embedded linux

### DISCUSSION THREAD

**Discussion by mershaywu77** [CppMicroServices/CppMicroServices#948] [General]
We are considering to use this framework in the future IoT project edge devices (yocto linux + ARM cortect M/A), any existing use cases/note/guide is appreciated.

**Comment by jeffdiclemente** [comment]
Hi @mershaywu77 , I recommend starting with [the getting started guide](https://docs.cppmicroservices.org/en/stable/index.html#general). The [docs](https://docs.cppmicroservices.org/en/stable/index.html#), in general, are a good reference resource as you explore cppmicroservices. 

Lastly, we maintain design docs for most pieces of functionality in the [rfc repository](https://github.com/CppMicroServices/rfcs) for reference. Once you have some familiarity with authoring and building cppmicroservices shared libraries, I recommend using [Declarative Services](https://github.com/CppMicroServices/rfcs/blob/master/text/0003-declarative-services.md) as the de-facto way of building and using cppmicroservices in your application.
